src/redux/countriesSlice.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] Still this issue not resolved, please enable all the linters and try to improve the code to fix the linters.